pkg/util/ipset/ipset.go

```diff
@@ -467,14 +467,8 @@ func validateHashFamily(family string) bool {
 func IsNotFoundError(err error) bool {
 	es := err.Error()
 	if strings.Contains(es, "does not exist") {
-		// set with the same name already exists
```

Comment:
It seems like other error codes also contain the "does not exist" string (https://github.com/Olipro/ipset/blob/c3ad11037681e46fb5f3096c5c824f3e3556522a/lib/errcode.c#L56-L58), should this be more specific?

Reply:
@andrewsykim 
`IsNotFoundError()` is called in the two places below.
https://github.com/kubernetes/kubernetes/blob/c2a4906152e67a45308f988aa2cb8b76b4503855/pkg/proxy/ipvs/ipset.go#L157-L158
https://github.com/kubernetes/kubernetes/blob/5373fa3f593a12309ff0435858991dd117bdb6ca/pkg/proxy/ipvs/proxier.go#L784-L786

IMO, the "does not exist" string should be enough to check error message from `DestroySet()` since we don't support other functions like `ipset swap` here.  
In the same way,  the "it's not added" string should be ok with `DelEntry()`